src/m1_facilitator/manifest/Manifest.ts

```diff
@@ -296,7 +276,11 @@ export default class Manifest {
       }
       const keystore = fs.readFileSync(acc.keystore_path).toString();
       const password = fs.readFileSync(acc.keystore_password_path).toString();
-      avatarAccounts[address] = new AvatarAccount(JSON.parse(keystore), password);
+      avatarAccounts[address] = AvatarAccount.load(
+        new Web3(''),
+        JSON.parse(keystore.trim()),
+        password.trim(),
```

Comment:
Don't use trim, a password can contain space as prefix or postfix.

Reply:
Since the passwords are being read from file, line terminator characters are being appended. This breaks load method as decrypt doesn't work.